vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -175,26 +179,9 @@ class OnboardingViewModel @AssistedInject constructor(
         }
     }
 
-    private fun handleSplashAction(resetConfig: Boolean, onboardingFlow: OnboardingFlow) {
-        if (resetConfig) {
-            loginConfig = null
-        }
-        setState { copy(onboardingFlow = onboardingFlow) }
-
-        return when (val config = loginConfig.toHomeserverConfig()) {
-            null -> continueToPageAfterSplash(onboardingFlow)
-            else -> startAuthenticationFlow(trigger = null, config, ServerType.Other)
-        }
-    }
-
-    private fun LoginConfig?.toHomeserverConfig(): HomeServerConnectionConfig? {
```

Comment:
no longer needed as the `SelectHomeserver` will read from the deeplink or default to matrix.org